client/src/main/java/com/distkv/asyncclient/DefaultAsyncClient.java

```diff
@@ -28,7 +28,13 @@ public DefaultAsyncClient(String serverAddress) {
             .build();
 
     rpcClient = new NettyClient(clientConfig);
-    rpcClient.open();
+
+    try {
+      rpcClient.open();
+    } catch (Exception e) {
```

Comment:
I think that catching **Exception** is not a good idea, although it is correct. And it is generic, catching **ConnectException** might be better.